docs/installations/install-11.md

```diff
@@ -133,34 +133,4 @@ redirect_from: /books/how-to-and-guides/page/clean-installing-windows-11
 
 You have successfully installed Windows 11.
 
-## Bypassing network requirements
-### Bypassing the network connection step
-If you reached the screen seen below, but don't want to connect to WiFi or are unable to connect and don't have the option for "I don't have internet", follow these steps: 
-     
-[![win11-network.png](/assets/install-11/win11-network.png)](/assets/install-11/win11-network.png)
-
-1. Press Shift + F10 on your keyboard. This will bring up a Command Prompt window. 
-
-2. Type in `OOBE\BYPASSNRO` and then press Enter. 
-
-[![win11-network_bypass.png](/assets/install-11/win11-network_bypass.png)](/assets/install-11/win11-network_bypass.png)
-
-The setup process will now restart. This will either give you the option for "I don't have internet", or skip to the local account creation screen. 
-You can continue setting up Windows 11. 
-
-### Bypassing the Microsoft account requirement
-If you reached the screen seen below, but don't want to log into a Microsoft account or don't have one, follow these steps: 
-
-[![win11-msft_acc_login.png](/assets/install-11/win11-msft_acc_login.png)](/assets/install-11/win11-msft_acc_login.png)
-
-1. Type in `no@thankyou.com` and press Enter. 
-
-[![win11-msft_acc_bypass.png](/assets/install-11/win11-msft_acc_bypass.png)](/assets/install-11/win11-msft_acc_bypass.png)
-
-2. Type in any password and press Enter. 
-
-[![win11-msft_acc_login_bypass2.png](/assets/install-11/win11-msft_acc_bypass2.png)](/assets/install-11/win11-msft_acc_bypass2.png)
-
-You will get an error like seen below, but it can be ignored and you can continue setting up Windows 11. 
-
-[![win11-msft_acc_login_bypass3.png](/assets/install-11/win11-msft_acc_bypass3.png)](/assets/install-11/win11-msft_acc_bypass3.png)
+If you need to setup Windows 11 without an internet connection or without a Microsoft account, follow this [article](/docs/guides/bypass-11.md).
```

Comment:
Can we do a section header above this, like in the LLCD article. Not sure what to call it though, maybe "## Advanced setup guides"